### PR TITLE
Initializing the project framework

### PR DESCRIPTION
**Pull request by yp945**

### CODE REVIEW

**Review by xuanyan0x7c7 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# olavm
-A pure Rust Olavm implementation.
+# Olavm
+Olavm's assembler and virtual machine cl
```
Comment (spartucus) [inline README.md:2]:
```suggestion
Olavm's assembler and virtual machine cl.
```

**Review by spartucus — CHANGES_REQUESTED** [review]

**Review by spartucus — APPROVED** [review]

**Comment by hongyuanyang-uu** [comment]
OlaVM命名后边统一一下

**Comment by hongyuanyang-uu** [comment]
Please use the name OlaVM for VM name

**Review by hongyuanyang-uu — APPROVED** [review]

**Comment by hongyuanyang-uu** [comment]
Readme need be modified.

**Pull request merged**